fem/bilinearform.cpp

```diff
@@ -1275,6 +1275,22 @@ void BilinearForm::Update(FiniteElementSpace *nfes)
    height = width = fes->GetVSize();
 
    if (ext) { ext->Update(); }
+   for (auto &integral : domain_integs)
+   {
+      integral->Update();
+   }
+   for (auto &integral : boundary_integs)
+   {
+      integral->Update();
+   }
+   for (auto &integral : interior_face_integs)
+   {
+      integral->Update();
+   }
+   for (auto &integral : boundary_face_integs)
+   {
+      integral->Update();
+   }
```

Comment:
Nitpick: maybe `integ` or `integrator` is a better name than `integral` (it's an integrator, not an integral).